sdks/python/apache_beam/runners/portability/sdk_container_builder.py

```diff
@@ -252,7 +252,13 @@ def _invoke_docker_build_and_push(self, container_image_name):
     build.steps = []
     step = cloudbuild.BuildStep()
     step.name = 'gcr.io/kaniko-project/executor:latest'
-    step.args = ['--destination=' + container_image_name, '--cache=true']
+    # Disable compression caching to allow for large images to be cached.
+    # See: https://github.com/GoogleContainerTools/kaniko/issues/1669
+    step.args = [
+        '--destination=' + container_image_name,
+        '--cache=true',
+        '--compressed-caching=false',
```

Comment:
Do we need to expose this option?

Reply:
my intuition is that probably not worth it.

Reply:
I don't think it's worth it either. Smaller images are small enough this shouldn't be an issue, but larger images are blocked by it. More configuration options confuse people in my opinion.